scripts/python/junglecatsoftware/awstoolkit/outputFormatters.py

```diff
@@ -5,18 +5,19 @@ def printBigHeader( author = 'Jungle Cat Software', package = 'AWS-Toolkit', fun
   firstLineLen = len( firstLine )
   secondLine = function
   secondLineLen = len( secondLine )
-
+  
```

Comment:
Is it common convention to do the spaces on blank lines? This wasn't repeated on line 12, so I'm wondering if this was either just auto-indenting from an IDE and if there is a general preference to either have spaces on the blank lines or keep them completely blank?